sdk/storage/CONTRIBUTING.md

```diff
@@ -18,28 +18,38 @@ The Azure Storage development team uses Visual Studio Code. However, any preferr
 
 ### Configuration
 
-The only step to configure testing is to set the appropriate environment variables. Create environment variables named "ACCOUNT_NAME", "ACCOUNT_KEY" or "ACCOUNT_SAS". The first two will be used for most requests. The "ACCOUNT_SAS" will only be used for tests in browsers.
+The only step to configure testing is to set the appropriate environment variables. Create environment variables named
+
+```bash
+ACCOUNT_NAME
+ACCOUNT_KEY
+ACCOUNT_SAS
+ACCOUNT_TOKEN
+TEST_MODE
+```
+
+The first two will be used for most requests. The "ACCOUNT_SAS" will only be used for tests in browsers.
 
 You can generate a valid account SAS from Azure portal or tools like Azure Storage Explorer. A SAS starts with "?". And if you are using Windows CMD, you may need quotes to escape special characters like following:
 
 ```bash
 set "ACCOUNT_SAS=<YOUR_SAS>"
 ```
 
-### Record and playback tests
+#### TEST_MODE
```

Comment:
>TEST_MODE [](start = 5, length = 9)

Is `TEST_MODE` now required, or has a default value of `live` if not specified?

Reply:
The default will be playback mode.
We have to explicitly set the TEST_MODE for `live` and `record` modes.

Reply:
The CI had been taken care of in this PR accordingly.

Reply:
ok I saw it now

---
In reply to: [364915021](https://github.com/Azure/azure-sdk-for-js/pull/6804#discussion_r364915021) [](ancestors = 364915021)